.spelling

```diff
@@ -299,6 +299,12 @@ CHANGELOG.md
 Toolset
 v6
 WiX
+Dockerfiles
+PowerShellWindowsDocker
```

Comment:
Can we keep the dictionary in alphabetical order per document?  This helps when reviewing to verify that you didn't add a duplicate word.